airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -1627,8 +1631,7 @@ public Stream<ConnectorBuilderProject> getConnectorBuilderProjectsByWorkspace(fi
 
     return database
         .query(ctx -> ctx
-            .select(CONNECTOR_BUILDER_PROJECT.ID, CONNECTOR_BUILDER_PROJECT.WORKSPACE_ID, CONNECTOR_BUILDER_PROJECT.NAME,
-                CONNECTOR_BUILDER_PROJECT.ACTOR_DEFINITION_ID)
+            .select(baseConnectorBuilderProjectColumns)
             .from(CONNECTOR_BUILDER_PROJECT)
             .where(matchByIds)
```

Comment:
We probably want to sort on something in this query (like create time or just name) to ensure the order will stay consistent